lib/maze/hooks/command_hooks.rb

```diff
@@ -0,0 +1,20 @@
+# Contains logic for the Cucumber hooks when in Command mode
+module Maze
+  class CommandHooks
```

Comment:
Would it make sense to call this `NoopHooks` or `NullHooks` or something along those lines as there's no logic here?

Reply:
I think it makes sense as it is.  Whilst it is noop at the moment, it could potentially change at any point.  Conceptually it's the hooks needed for Command mode, they just happen to be empty at the moment - so the abstraction feels right to me.